README.md

```diff
@@ -1,52 +1,124 @@
 
-ApostropheCMS is Node.js all the way through, which means it can be [extended](https://apostrophecms.com/extend) in all sorts of interesting ways. Here's a list of some of our favorite modules:
+  [apostrophe-headless](https://github.com/apostrophecms/apostrophe-headless) - Add REST APIs and power your React / Vue Native / etc. apps with a headless CMS.
+  [apostrophe-blog](https://github.com/apostrophecms/apostrophe-blog) - Everything you need to put a blog on your site.
+  [apostrophe-events](https://github.com/apostrophecms/apostrophe-events) - Manage upcoming events.
+  [apostrophe-workflow](https://github.com/apostrophecms/apostrophe-workflow) - Add powerful localization capabilities to your project.
+  [apostrophe-passport](https://github.com/apostrophecms/apostrophe-passport) - Authenticate via Twitter, Facebook, Github and more.
+  [apostrophe-saml](https://github.com/apostrophecms/apostrophe-saml) - Authenticate via Shibboleth, as well as corporate SAML environments like Salesforce.
+  [apostrophe-places](https://github.com/apostrophecms/apostrophe-places) - Manage geotagged content and display it on a map.
+  [apostrophe-redirects](https://github.com/apostrophecms/apostrophe-redirects) - An easy redirects module.
 
-* [apostrophe-headless](https://github.com/apostrophecms/apostrophe-headless) - Add REST APIs and power your React / Vue / Native / etc. apps with a headless CMS.
-* [apostrophe-blog](https://github.com/apostrophecms/apostrophe-blog) - Everything you need to put a blog on your site.
-* [apostrophe-events](https://github.com/apostrophecms/apostrophe-events) - Manage upcoming events.
-* [apostrophe-workflow](https://github.com/apostrophecms/apostrophe-workflow) - Add powerful localization capabilities to your project.
-* [apostrophe-passport](https://github.com/apostrophecms/apostrophe-passport) - Authenticate via Twitter, Facebook, Github and more.
-* [apostrophe-saml](https://github.com/apostrophecms/apostrophe-saml) - Authenticate via Shibboleth, as well as corporate SAML environments like Salesforce.
-* [apostrophe-places](https://github.com/apostrophecms/apostrophe-places) - Manage geotagged content and display it on a map.
-* [apostrophe-redirects](https://github.com/apostrophecms/apostrophe-redirects) - An easy redirects module.
 
-## Additional Resources
-We've got some great [open-source tools and how-to articles](https://docs.apostrophecms.org/apostrophe/howtos/howtos) that can help you out deploying, managing and maintaining ApostropheCMS sites.
 
-## Community
+<!-- CONTRIBUTING -->
+## Contributing
 
-**Report bugs and submit pull requests** via [github](https://github.com/apostrophecms/apostrophe/issues).
+We eagerly welcome open source contributions. Before submitting a PR, please read through our [Contribution Guide](https://github.com/apostrophecms/apostrophe/blob/main/CODE_OF_CONDUCT.md)
```

Comment:
We have a contributing.md file. Let's link to that and not worry about instructions for PRs. There are other/easier ways to start contributing.

Reply:
Yes please. Let's not maintain two of anything.